package.json

```diff
@@ -64,7 +64,7 @@
       },
       {
         "permissionName": "ui-finance.fiscal-year.view",
-        "displayName": "Fiscal year: View",
+        "displayName": "Finance: View fiscal year",
```

Comment:
but changing this means that translations doesn't work correctly. Currently platform uses translations instead of displayName